www/havm10/ukol2.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="cs">
+
+<head>
+    <meta charset="utf-8" />
+    <title>VŠE-pro-VŠE</title>
+</head>
+
+<body>
+    <header>
+        <span>VŠE-pro-VŠE</span>
+        <nav>
+            <a href="">Domovská stránka</a>
+            <br>
+            <a href="">Katalog produktů</a>
+            <br>
+            <a href="">Obchodní podmínky</a>
+            <br>
+            <a href="">Způsoby dodání</a>
+        </nav>
+    </header>
+    <h1>VŠE-pro-VŠE, vše pro vaše studium</h1>
+    <ul>
+        <li>Lorem ipsum dolor sit amet consectetur adipisicing elit. Reiciendis exercitationem eius impedit omnis
```

Comment:
Pěkný, ale produkt by měl mít kromě popisu i nějaký název, a pak ještě nějaké tlačítko, kam se proklikne dal.

Reply:
Vyřešil sem to tímto způsobem, tak snad to bude OK